### PR TITLE
:recycle: Replace buttons on workspace

### PR DESCRIPTION
**Pull request by luisddm**
- Replace icon buttons in workspace https://tree.taiga.io/project/penpot/task/8913

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 1]
```diff
@@ -759,9 +760,10 @@
                              {:value :percent :label "%"}]
                    :on-change #(set-column-type type index %)}]]]
 
-     [:button {:class (stl/css :remove-track-btn)
-               :on-click #(remove-element type index)}
-      i/remove-icon]]))
+     [:> icon-button* {:variant "ghost"
+                       :aria-label (tr "workspace.shape.menu.delete")
+                       :on-click #(remove-element type index)
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: Remove this partial so code is cleaner.

File: frontend/src/app/main/ui/comments.cljs [hunk 2]
```diff
@@ -335,9 +336,10 @@
...
        (when (= (:id profile) (:id owner))
-         [:div {:class (stl/css :options)
-                :on-click on-toggle-options}
-          i/menu])]
+         [:> icon-button* {:variant "ghost"
+                           :aria-label (tr "labels.options")
+                           :on-click on-toggle-options
+                           :icon "menu"}])]
 
       [:div {:class (stl/css :content)}
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/comments.cljs:344]: Remove the scss of this button if posible

File: frontend/src/app/main/ui/workspace/sidebar/assets/components.cljs [hunk 3]
```diff
@@ -527,9 +528,10 @@
...
 
       (when (and components-v2 (not read-only?) local?)
-        [:div {:on-click add-component
-               :class (stl/css :add-component)}
-         i/add
+        [:> icon-button* {:variant "ghost"
+                          :aria-label (tr "workspace.assets.components.add-component")
+                          :on-click add-component
+                          :icon "add"}
          [:& file-uploader {:accept cm/str-image-types
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/workspace/sidebar/assets/components.cljs:535]: Remove css of this button if posible.

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs [hunk 4]
```diff
@@ -172,19 +172,26 @@
...
+         [:> icon-button* {:variant "ghost"
+                           :aria-label (tr "workspace.shape.menu.lock")
+                           :on-click handle-set-blocked
+                           :icon "unlock"}]
 
          :else
-         [:button {:on-click handle-set-unblocked
-                   :class (stl/css-case :lock-btn true
-                                        :locked blocked?)} i/lock])]]]))
+         [:> icon-button* {:variant "ghost"
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs:194]: This button changed where locked, Did you implement this behavior?

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/exports.cljs [hunk 5]
```diff
@@ -233,9 +235,10 @@
                          :on-change on-suffix-change
                          :on-key-down manage-key-down}]]]
 
-              [:button {:class (stl/css :action-btn)
-                        :on-click (partial delete-export index)}
-               i/remove-icon]])])
+              [:> icon-button* {:variant "ghost"
+                                :aria-label (tr "workspace.options.export.remove-export")
+                                :on-click (partial delete-export index)
```
Comment (EvaMarco) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/exports.cljs]: Remove this Partial

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs (hunk 4 above)
Comment (luisddm) [reply frontend/src/app/main/ui/workspace/sidebar/options/menus/layer.cljs:194]: With the old code, the button adds the class "locked" to itself when the element is locked, so it seems it was expected to get a new appearance, not only an icon toggle. The problem is that the mentioned class does not exist, so effectively it does nothing and makes the condition unnecessary.

**Pull request merged**